packages/projection/src/operators/logProjectionProgress.ts

```diff
@@ -29,20 +77,8 @@ export const logProjectionProgress =
                 eventType === ChainSyncEventType.RollForward ? 'RollForward' : 'RollBackward'
               } ${pointDescription(header)}`
             );
-          } else if (numEvt % logFrequency === 0 && tip !== 'origin') {
-            const syncPercentage = ((header.blockNo * 100) / tip.blockNo).toFixed(2);
-            const now = Date.now();
-            const currentSpeed = Math.round(logFrequency / ((now - lastLogAt) / 1000));
-            lastLogAt = now;
-            const overallSpeedPerMs = numEvt / (now - startedAt);
-            const overallSpeed = Math.round(overallSpeedPerMs * 1000);
-            const eta = new Date(now + (tip.blockNo - header.blockNo) / overallSpeedPerMs);
-            logger.info(
-              `Initializing ${syncPercentage}% at block #${
-                header.blockNo
-              }. Speed: ${currentSpeed}bps (avg ${overallSpeed}bps). ETA: ${eta.toISOString()}`
-            );
-          }
+          } else if (numEvt % logFrequency === 0 && tip !== 'origin')
+            logSyncLine({ blocksTime, header, logger, numEvt, startedAt, tip });
         }),
         finalize(() => logger.info('Stopped'))
```

Comment:
Would be useful to log the total projection time. exitAtBlockNo might not be a multiple of any interval, so we probably can't rely on those logs to determine total time.
```suggestion
        finalize(() => logger.info(`Stopped after ${totalTime}`))
```

Reply:
9eef8f59e should solve this